packages/origin-ui-core/src/components/Form/FormCountrySelect.tsx

```diff
@@ -9,13 +9,24 @@ interface IFormCountrySelectProps {
     className?: string;
     required?: boolean;
     disabled?: boolean;
+    isoFormat?: boolean;
 }
 
 const COUNTRY_OPTIONS = Countries.map((country) => ({
     value: country.id,
     label: country.name
 }));
 
+const COUNTRY_OPTIONS_ISO = Countries.map((country) => ({
+    value: country.id,
+    label: country.name,
+    code: country.code
+}));
+
 export function FormCountrySelect(props: IFormCountrySelectProps) {
-    return <FormSelect options={COUNTRY_OPTIONS} {...props} />;
+    if (props.isoFormat) {
+        return <FormSelect options={COUNTRY_OPTIONS_ISO} {...props} />;
+    } else {
+        return <FormSelect options={COUNTRY_OPTIONS} {...props} />;
+    }
```

Comment:
Can be a one line: 
`return <FormSelect options={props.isoFormat ? COUNTRY_OPTIONS_ISO : COUNTRY_OPTIONS} {...props} />;`